samples/azure-quantum/grover/Grover.ipynb

```diff
@@ -0,0 +1,710 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
```

Comment:
We may want to add `nteract` to cspell.json to minimize false positives here.

Reply:
Done!